config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+wAguXGXv2Th1PvfGEDCElX0DoK63PD0/6/gJKT3SBb/Cf088PpZVaL9pLYoNg21zBSGwaLgqWhhu7TfamPXIZ9VIRxVpZr9m+8CNBFnunFk3R6ClDEeAmbolVByk3BAKpopaKk/o2E6Sbc6Y3gsKcMmNEo5vpIi700qbzCJErfgnQVKQRsY5F1vZnCeoZ2rEX5b8v15uwYjMaHnMc1xZIQULcmw/XVrBGe2cD9MA4qXz1/le47/pbkGu0z3piAI3Ner3+ZvnC5GOwpNlR0PeZkdJA5p9tYHcCg+VTQ/lFUiJJa+2VBQt3hhGEoML0iz/ucMh6/N7P5Apj5WW1aHm5yvcFBFmxKCMnaC6OO2E3Dawu0ND4SMIZufv9T5Q8ZeCyVc+Wli96x/uVmhAmIEd394mFTbZhUBnyQsC--Bub744Vr/LSCEwT+--jccmVXytoCYw1VpVLMrDZg==
```

Comment:
These are encoded credentials, right? You are not checking in sensitive data into a git repository, right?

Reply:
I mean encrypted, not encoded

Reply:
without the master key, which is ignored by git the credentials can't be accessed. it's generally speaking safe to version this file.